### ISSUE TITLE
Disable the "dislike" feature instance wide

### ISSUE THREAD

**Issue by atasingorneq** [friendica/friendica#13007]
### Describe the feature you'd like

The ability for admins to disable the "dislike" feature system-wide.

Giving a post or comment a "dislike" can lead to misunderstandings more easily than giving a "like".

Example:

If I share a newspaper article in a post and a "dislike" is given, I can interpret that in different ways. Does the "dislike" apply to the statement in the newspaper article or does the "dislike" apply to the fact that I shared the article.

### Additional context

Two threads about this feature:

- [https://friendica.xyz/display/adf174d5-4664-36a8-06f3-1b0425925893](https://friendica.xyz/display/adf174d5-4664-36a8-06f3-1b0425925893)
- [https://friendica.mrpetovan.com/display/735a2029-755e-41ef-b0f0-974890376709](https://friendica.mrpetovan.com/display/735a2029-755e-41ef-b0f0-974890376709)

**Comment by MrPetovan** [comment]
Of course I support this, but it isn't straightforward, three years ago we had a poll and the result was to make it user-based. Putting back in the additional features would essentially revert this feature to its state three years ago before the poll was conducted.

I'm not experienced in community governance, but I'm not comfortable going forward with this change right away even if it was suggested by another member of the community.

Should we have another poll?

**Comment by loma-one** [comment]
No matter how it is solved, there is a function behind it that is distributed in the Fediverse, e.g. to Lemmy. This should be preserved. If I understood correctly, it was not about the function, but about the symbol. These are two different things.

**Comment by annando** [comment]
@loma-one yeah. Things like this are the reason why I think that this should be a user and not an admin setting.

**Comment by tek-aevl** [comment]
I  like the dislike option, I use it. If we had more reaction options that would be interesting.

**Comment by MrPetovan** [comment]
> @loma-one yeah. Things like this are the reason why I think that this should be a user and not an admin setting.

Putting it back in "Additional Features" gives agency to both. Admins can set a node-wide policy (always enabled, always disabled, user-controlled with enabled by default, user-controlled with disabled by default) and users may be able to choose for themselves.

**Comment by loma-one** [comment]
From my point of view, it would make more sense to create instance-wide settings for the functions: "Scroll endless" and "Stay local". 

Most users have problems finding the function. 
Friendica seems unattractive with the current pre-installations, which leads to many users preferring other platforms.

They are used to other platforms and expect content to be reloaded and to stay within their application framework and not be bounced to other platforms.

**Comment by annando** [comment]
@loma-one please create a separate issue for that, so that we don't lose it out of the focus.

**Comment by AndyHee** [comment]
> community governance

I think, the following criteria should be met:

- Signifiant renewal of interest in the matter 
- Broad support to revisit the matter
- Considerable amount of time passed from the last decision
- Willingness to use the very same method for reaching a new decision that was previously used (e.g. voting, consensus)

**Comment by davidspring** [comment]
I also feel that there should be a check box for Friendica instance administrators to turn off Dislike. The word Dislike is too ambiguous, negative and critical. It can lead to a confusing double negative that does not occur with the word Like. 

I am a big fan of democracy and voting. However, as Tom Jefferson once observed, it is also important to not let the tyranny of the majority trample on the rights of the minority – especially when those actually voting represent less than one percent of all of the users. 

While we are waiting for this feature to be added, I want to let site admins know that it is relatively easy for a site admin to hide the word and icon for Dislike. Go to view/themes/frio/css and open the style.css file. At Line 2355, just before Notification Badges, enter a new line. Then copy paste: 

[id^="dislike-"] {display: none !important;}

In addition, at the blank Line 3716, enter a new line. Then copy paste: 

.fa-thumbs-down .button-links .button-likes {display: none !important;}
.wall-item-actions a, .wall-item-actions button
.wall-item-actions..fa-thumbs-down {display: none !important;}

These two changes to the Frio theme CSS file will hide the Dislike function on all posts on the forum(s) they administer.

**Comment by marcusxss** [comment]
There are multiple factors resulting in Facebook taking a massive downward turn. Things like: 
- almost no moderation compared to the amount of content, even when things obviously goes against their community standards
- no user verification, so those banned/blocked come back a few minutes later with another account making moderation (and things like avoiding stalkers) extra challenging 
- related to the above: allowing duplicate accounts
- allowing anonymity - awesome in theory but apparently some people can't handle it
- the ads
- harmful algorithms that amplify controversy/disagreement - I think this can really do a lot of damage over time

I'm not entirely sure what the solution is regarding the above, because "fixing" them can also have extensive drawbacks. some people need anonymity and most can still act decently to others even when anonymous, fx.

...BUT another thing I think has contributed to it, is the dislikes and laugh reacts.

I feel like those can easily lead to a more hostile atmosphere, and people brigading to down-vote things fx. to be mean, for political reasons etc., which is a lot easier than the more constructive solution, of actually voicing the disagreement in the comments.

...so yeah, I support the ability for instances to remove dislikes server-wide.
but even being able to set the default for new users would be great, as it seems the current default is to show them.

**Comment by loma-one** [comment]
This feature is disabled by default for new accounts. Users must therefore make a conscious decision to enable this feature in order to use it in environments such as Lemmy, MBIN, etc. It is also part of the AP protocol.
